plugin/Controller/AdminViewController.php

```diff
@@ -427,19 +428,29 @@ public function generalAdminNoticeSEO() {
 		$urlOnofficeSetting = admin_url().'admin.php?page=onoffice-settings';
 		$nameOnofficeSetting = esc_html__('onOffice plugin settings','onoffice-for-wp-websites');
 		$pluginOnofficeSetting = sprintf("<a href='%s' target='_blank' rel='noopener'>%s</a>", $urlOnofficeSetting,$nameOnofficeSetting);
+		$pluginSEO = [];
+		if (count(array_intersect(["wpseo/wp-seo.php"], get_option("active_plugins"))) > 0) {
+			$pluginSEO['wp_seo'] = 'wpSEO';
+		}
+		if (count(array_intersect(["seo-by-rank-math/rank-math.php"], get_option("active_plugins"))) > 0) {
+			$pluginSEO['rank_math_seo'] = 'Rank Math SEO';
+		}
+		if (count(array_intersect(["wordpress-seo/wp-seo.php"], get_option("active_plugins"))) > 0) {
+			$pluginSEO['yoast_seo'] = 'Yoast SEO';
+		}
```

Comment:
Would it make sense to extract this code into a function, since it is also used in https://github.com/onOfficeGmbH/oo-wp-plugin/pull/118/files/2c2659d9cd2c438696fa21e968c49a6863948678..3bb7947349d4816aecc4add7a78a875f5106a143?file-filters%5B%5D=.css&file-filters%5B%5D=.json&file-filters%5B%5D=.lock&file-filters%5B%5D=.php&show-viewed-files=true#diff-c829b20424fa6035b35f0a1df38bda120f735f3eec5c0a2b5aa4a1004f06b973R180-R188?

Reply:
sorry, i still working PR !